source/math/Quaternion.ooc

```diff
@@ -260,6 +262,105 @@ Quaternion: cover {
 		}
 		result
 	}
+	weightedQuaternionMean: static func (quaternions: VectorList<This>, weights: Float[]) -> This {
```

Comment:
Actually, there is no good reason... I will change `Float[]` to `FloatVectorList`. Then I can also get rid of the function `_normalizeFloatArrayForQuest` and just use `weights / weights sum`. Better?